packages/gui/src/index.tsx

```diff
@@ -13,6 +13,14 @@ import Container from "./Container";
 import { ModeProvider } from "./contexts/ModeContext";
 
 import "./css/index.css";
+import { StyledEngineProvider, Theme } from "@mui/material/styles";
+
+
+declare module '@mui/styles/defaultTheme' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
+}
+
```

Comment:
Is this needed? I don't see `@mui/styles/defaultTheme` being imported here.

Reply:
It's needed for the Theme [here](https://github.com/stateful/vscode-marquee/pull/45/files/303f7c5bfa0f49aff2a0e419deaa838075d610de#diff-18ebd9b5f02698ff1a85ffa4242b483de344458bb9a4cac497f4907fc31fabb0R16), which is a re-export. 

You very reasonably could ask, "then why not declare the interface on `@mui/material/styles`?". 

That's because under the hood that interface is also the one typing the `makeStyles`, e.g. [here](https://github.com/stateful/vscode-marquee/pull/45/files/303f7c5bfa0f49aff2a0e419deaa838075d610de#diff-8a2b1309a106ab83f7f4c3de8899b8bf3161e833e4b2492097353dd36ce74c3fR18).

You can read a bit more about why this is necessary in the first place in the [MUI upgrading docs.](https://mui.com/guides/migration-v4/#types-property-quot-palette-quot-quot-spacing-quot-does-not-exist-on-type-defaulttheme)